src/common/containers/CitationListContainer.jsx

```diff
@@ -49,22 +51,33 @@ class CitationListContainer extends Component {
     dispatch(fetchCitations(pidType, recordId, { page, pageSize: PAGE_SIZE }));
   }
 
-  render() {
-    const { loading, total, citations } = this.props;
+  renderError() {
+    const { error } = this.props;
+    return <ErrorAlert message={error.get('message')} />;
+  }
+
+  renderList() {
+    const { total, citations } = this.props;
     const { page } = this.state;
+    return (
+      total > 0 && (
+        <ListWithPagination
+          renderItem={CitationListContainer.renderCitationItem}
+          pageItems={citations}
+          onPageChange={this.onPageChange}
+          total={total}
+          page={page}
+          pageSize={PAGE_SIZE}
+        />
+      )
+    );
+  }
+
+  render() {
+    const { loading, error } = this.props;
     return (
       <ContentBox loading={loading}>
-        {total > 0 && (
-          <ListWithPagination
-            renderItem={CitationListContainer.renderCitationItem}
-            pageItems={citations}
-            onPageChange={this.onPageChange}
-            total={total}
-            loading={loading}
-            page={page}
-            pageSize={PAGE_SIZE}
-          />
-        )}
+        {error ? this.renderError() : this.renderList()}
```

Comment:
It would be nice to create a component which handles the errors i.e. ``ErrorOrChildren`` or smth

Reply:
if we wanted to display something else the component needs to have renderError method so it will become like `<If condition={...} whenTrue={...} whenFalse={...} />`

Reply:
But I guess for now `ErrorAlertOrChildren` would be OK anyway.